example/docker-compose.yml

```diff
@@ -28,6 +28,8 @@ services:
 
     db:
         image: mariadb:10 # or mysql:5.7
+        ports:
+            - 3307:3306 # change ip if required
```

Comment:
Remove this since it's out of scope for this PR.

Reply:
Done